Marlin/src/module/temperature.cpp

```diff
@@ -44,7 +44,39 @@
   #include "../lcd/extui/ui_api.h"
 #endif
 
-#if MAX6675_0_IS_MAX31865 || MAX6675_1_IS_MAX31865
+#if MAX6675_HAS_MAX31855
```

Comment:
this macro name is a bit confusing to me.... how MAX6675 _has_ MAX31855? Did you mean something like `HAS_MAX6675_OR_MAX31855`?

Reply:
The temperature.cpp file does all processing for SPI boards in Temperature::read_max6675().   Therefore, I help to the pattern set before me to use HAS_MAX6675_??? for anything that has to do with the read_max6675 routine

Reply:
That don't explain why one uses the other. What you mean by 6675 uses 13855?

Reply:
My understanding was that everything was built on top of the MAX6675 code. I am assuming they reused the old code to start the new stuff.  So when the MAX31865, was added they used names like MAX6675_uses_MAX31865 or MAX6675_IS_MAX31865.   So I just kept to that pattern.  What do you prefer?